terraform/environments/digital-prison-reporting/locals.tf

```diff
@@ -297,6 +298,10 @@ locals {
   # CW Insights
   enable_cw_insights = local.application_data.accounts[local.environment].setup_cw_insights
 
+  # Setup Atena Workgroups 
```

Comment:
Was being nosy as also trying to setup an Athena workgroup, noticed this line had a mis-spelling

Reply:
@luke-a-williams  - thanks for being Nosy - 

great catch ;)